<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dlg_AddStu</class>
 <widget class="QDialog" name="Dlg_AddStu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>304</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加与修改</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>194</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QPushButton" name="btn_cancel">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="btn_save">
      <property name="text">
       <string>保存</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>52</y>
     <width>30</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>性别</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_phone">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>114</y>
     <width>259</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_sno">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>196</y>
     <width>259</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>93</y>
     <width>30</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>电话</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>175</y>
     <width>30</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>学号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_name">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>32</y>
     <width>259</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_sex">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>73</y>
     <width>259</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="sb_age">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>155</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>30</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>姓名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>134</y>
     <width>30</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>年龄</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
